### ISSUE TITLE
Implement voting for answers

### ISSUE THREAD

**Issue by gooddaytoday** [Mayank0255/Stackoverflow-Clone#12]
As i see, it's need to:

1. Add vote counting for answers in database
2. Add handling of voting on client and saving to DB on server.

**Comment by Mayank0255** [comment]
Hi @gooddaytoday 

Sorry, it took me a lot of time to go over this and finally I am ready to remove the restriction on the project and make it fully open source 🎉🎉

So, would like to implement this feature on the repo? I can just reopen it and assign this to you then.

**Comment by alceil** [comment]
Is anobody working on this issue?

**Comment by Mayank0255** [comment]
@alceil Sure, go ahead

**Comment by alceil** [comment]
Can anyone brief this issue a bit? handling of upvoting can be done in the front end. But to handle the db part I need to add changes in the back end too right? @Mayank0255

**Comment by Mayank0255** [comment]
@alceil yes changes in backend will also be required as there isn't any table there at the moment which keeps track of the votes.

And to begin with this I think adding voting to the questions only is satisfactory at the moment, later on we can go towards adding voting to answers as well. So, connecting the voting table with the post/question table would be required only, like suppose using `postId` as the foreign key

**Comment by Mayank0255** [comment]
@alceil you can refer to this https://data.stackexchange.com/stackoverflow/query/new

This is the querying website of the original stackoverflow which helps you test out SQL queries  on there database and on the right side it also displays all the schemas.

You can check out how they have implemented voting in the backend and how is there architecture to refer.

If you get stuck anywhere or have any other queries then I am always there here or reach out to me on my discord Shadowsaver#9665

**Comment by alceil** [comment]
Sure. Thanks ;)

**Comment by alceil** [comment]
I'm facing some problems when i set up the project locally. Can you help me out? And I need to run the backend too in another terminal right for the APIs to work.?

**Comment by Mayank0255** [comment]
@alceil yes you need to run both the servers, backend first then frontend

**Comment by alceil** [comment]
okay got it

**Comment by joshnho** [comment]
Is this issue being worked on ?

**Comment by Mayank0255** [comment]
@joshnho 
No, it isnt. 
Are you interested in picking it up?

**Comment by joshnho** [comment]
Yes I would like to work on it starting tonight

**Comment by Mayank0255** [comment]
@joshnho 
Sorry for the late reply, but ya go ahead.
Let me know if you face any issues

**Comment by Mayank0255** [comment]
@joshnho 
Hoping everything is going fine with the task, let me know if you get stuck anywhere

**Comment by ayushnau** [comment]
@Mayank0255 can I work on this issue.
Please assign this to me.

**Comment by Mayank0255** [comment]
@Ayush2020012016 
Sure, go ahead

Any idea that how much time will it take you to do this?

**Comment by Mayank0255** [comment]
@Ayush2020012016 how's the progress on this?
